### ISSUE TITLE
NullReferenceException when code uses ConfigurationManager

### ISSUE THREAD

**Issue by AmadeusW** [CodeConnect/AliveFeedback#13]
When working with a controller on an ASP.MVC web server, we get `NullReferenceException` at runtime

```
Runtime exception System.NullReferenceException: Object reference not set to an instance of an object.
   at CodeConnect.OrangeServer.Models.LicensingDataContext..ctor()
       ...
```

The constructor that throws the exception:

``` csharp
        public LicensingDataContext() : 
                base(global::System.Configuration.ConfigurationManager.ConnectionStrings["ComeAliveDBConnectionString"].ConnectionString, mappingSource)
        {
            OnCreated();
        }
```

It seems that the exception is caused by missing `System.Configuration.ConfigurationManager`
The connection strings are defined in app.config for the instrumented project, and Alive needs to have access to this configuration.

**Comment by JoshVarty** [comment]
Looks like we simply need to find the `app.config` and `web.config` files and include them in the AppDomain when we're running.

See: http://stackoverflow.com/questions/10906946/access-to-config-files-in-roslyn-repl
And: http://stackoverflow.com/questions/11993546/app-config-for-dynamically-loaded-assemblies

**Comment by AmadeusW** [comment]
Fixed in [0.2.3](https://github.com/CodeConnect/AliveFeedback/wiki/Changelog-0.2.3)
